### PR TITLE
Update/run ci script

### PR DESCRIPTION
**Pull request by adrianfusco**
I'd like to propose an improvement to the `run_ci` script.

- Change shebang for portability.
- Add shell options in the script. If there's an error it will automatically exit. e.g:
```bash
[2023-02-23 23:03:58] {afuscoar@afuscoar} (~/.Personal/Projects/devops-exercises) (update/run_ci-script)$ -> bash scripts/run_ci.sh 
/home/afuscoar/.Personal/Projects/devops-exercises/scripts/../README.md failed
Missing opening detail tag round line 4
[2023-02-23 23:03:59] {afuscoar@afuscoar} (~/.Personal/Projects/devops-exercises) (update/run_ci-script)$ -> echo $?
1
```
- Change the test execution for better readability.

### CODE REVIEW

**Comment by apiviuw** [comment]
Selesai

**Comment by bregman-arie** [comment]
Great change @adrianfusco
Thank you!

File: scripts/run_ci.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+
+MD_FILES=$(find ${PROJECT_DIR} -name "*.md" -not -path "${PROJECT_DIR}/tests/*")
+
+for file in ${MD_FILES[@]}; do
+   python ${PROJECT_DIR}/tests/syntax_lint.py ${file} > /dev/null
+done
+
+echo "- Syntax lint tests on MD files passed sucessfully"
+
+flake8 --max-line-length=100 . && echo "- PEP8 Passed"
```
Comment (RakeshSunda) [inline scripts/run_ci.sh:15]: hero4

**Review by RakeshSunda — COMMENTED** [review]
ok

**Comment by sriramsowmithri9807** [comment]
Proposed Improvements to` run_ci.sh`
Suggested Changes

Update Shebang:
Change the shebang to `#!/usr/bin/env` bash for better portability.

Add Error Handling:
Include the following options at the start of the script:
```
bash
set -euo pipefail
```
This will make the script exit on errors and handle unset variables properly.

Enhance Readability:
Refactor the script to use functions for better organization:
```
bash
run_tests() {
    echo "Running tests..."
    ./run_tests.sh 
}

check_results() {
    echo "Checking results..."
}

main() {
    run_tests
    check_results
}

main "$@"
```

**Pull request merged**